compiler-rt/lib/sanitizer_common/sanitizer_allocator_primary64.h

```diff
@@ -113,6 +113,23 @@ class SizeClassAllocator64 {
   // ~(uptr)0.
   void Init(s32 release_to_os_interval_ms, uptr heap_start = 0) {
     uptr TotalSpaceSize = kSpaceSize + AdditionalSize();
+
+    uptr MaxAddr = GetMaxUserVirtualAddress();
+    // VReport does not call the sanitizer allocator.
+    VReport(3, "Max user virtual address: 0x%zx\n", MaxAddr);
+    VReport(3, "Total space size for primary allocator: 0x%zx\n",
+            TotalSpaceSize);
+    // TODO: hypothetical edge case: on >48-bit VMA systems, Linux by default
```

Comment:
it's not immediately clear what there is TO DO

Reply:
There is nothing to do in the immediate or even medium-term future. But suppose it's the year 2800: 256TB heaps are commonplace, this warning message keeps popping up. and you get woken from cryogenic sleep because you are the last-known maintainer of Android sanitizers; you'll be glad the TODO is there to hint that the check needs revising.

Reply:
I've revised it